modules/imap/hm-ews.php

```diff
@@ -840,6 +860,25 @@ public function get_mime_message_by_id($itemId) {
         return $parser->parse($content, false);
     }
 
+    protected function get_item_id_base64($itemId) {
```

Comment:
I understand the desire to use this function but it is masking other problems. If we expect hex internal format, we should receive hex internal format. If we receive base64 instead, there is a problem somewhere else and we shouldn't mask it.